javascript/overlayvideo.js

```diff
@@ -48,7 +48,9 @@
         },
         events: {
           'onReady': 'onPlayerReady'
-        }
+        },
+        host: 'https://www.youtube.com',
```

Comment:
@phalkunz I remember adding `host` and `origin` but would you be able to check if it works without these? I may have just had some sort of chrome extension in my machine that could have been messing it up.

If it works without these, then we can take it off. Alternatively, if we need it, then we need to see if its only `youtube` URL's they want? Could they embed from other sources i.e. Vimeo?